frontend/src/components/ParticipantForm.js

```diff
@@ -64,7 +69,7 @@ function ParticipantForm(props) {
         />
```

Comment:
Did you want to add the disabled attribute to the dob field as well?

Reply:
oops yes i guess so!